python/lsst/ts/MTMount/telemetry_client.py

```diff
@@ -81,6 +81,17 @@ def __repr__(self):
 
 class TelemetryClient:
     on_drive_states = frozenset(("Standstill", "Discrete Motion", "Stopping"))
+    """Read telemetry from the low-level controller and write as SAL messages.
```

Comment:
Probably better as (missing subject?)

Read telemetry from the low-level controller and write **it** as SAL messages.

Reply:
The problem was getting it to fit. But I've tried a variant that works.